providers/ms365/resources/ms365.lr.manifest.yaml

```diff
@@ -258,7 +258,7 @@ resources:
       safeAttachmentPolicy: {}
       safeLinksPolicy: {}
       sharedMailboxes:
-        min_mondoo_version: latest
+        min_mondoo_version: 9.2.13
```

Comment:
In the future these should be the major version since any 9.x client can download this updated provider